content/en/serverless/azure_app_services.md

```diff
@@ -10,187 +10,142 @@ further_reading:
 ---
-<div class="alert alert-warning"> This service is in public beta. If you have any feedback, contact <a href="/help">Datadog support</a>. During the beta period, the use of this extension is not metered for billing purposes.</div>
 
 ## Overview
 
-Microsoft Azure App Services is a group of serverless resources that enable you to build and host web apps, mobile back ends, event-driven functions, and RESTful APIs without managing infrastructure. It can host workloads of all sizes and offers auto-scaling and high availability options.
+Microsoft [Azure App Services][1] is a group of serverless resources that enable you to build and host web apps, mobile back ends, event-driven functions, and RESTful APIs without managing infrastructure. It can host workloads of all sizes and offers auto-scaling and high availability options.
 
 Datadog provides monitoring capabilities for all Azure App Services resource types:
 
-- Azure Monitor metrics for [Apps][1] and [Functions][2] using the [Azure Integration][1].
+- Azure Monitor metrics for [Apps][2] and [Functions][3] using the [Azure Integration][2].
 - Custom metrics can be submitted using the API.
-- [Resource logs][3] can be submitted using [Event Hub][4].
+- [Resource logs][4] can be submitted using [Event Hub][5].
 
-The Datadog extension for Azure App Services provides additional monitoring capabilities for [Azure Web Apps][5]. This support includes:
+The Datadog extension for Azure App Services provides additional monitoring capabilities for [Azure Web Apps][6]. This support includes:
 
 - Full distributed APM tracing using automatic instrumentation.
 - Support for manual APM instrumentation to customize spans.
 - `Trace_ID` injection into application logs.
-- Support for submitting custom metrics using [DogStatsD][6].
+- Support for submitting custom metrics using [DogStatsD][7].
 
 ## Setup
 
 ### Requirements
 
-If you haven't already, set up the [Microsoft Azure integration][7] first.
+1. If you haven't already, set up the [Microsoft Azure integration][8] first.
 
-The Datadog .NET APM extension supports the following .NET runtimes in both x64 and x86 architectures when running on Windows instances (AAS does not yet support extensions on Linux). For more details about automatically instrumented libraries, see the [Tracer documentation][8].
+2. The Datadog .NET APM extension supports the following .NET runtimes in both x64 and x86 architectures when running on Windows instances (AAS does not yet support extensions on Linux). For more details about automatically instrumented libraries, see the [Tracer documentation][9].
 
-- .NET Framework 4.7 and later
-- .NET Core 2.1
-- .NET Core 2.2 (Microsoft support ended 2019-12-23)
-- .NET Core 3.0 (Microsoft support ended 2020-03-03)
-- .NET Core 3.1
+    - .NET Framework 4.7 and later
+    - .NET Core 2.1
+    - .NET Core 2.2 (Microsoft support ended 2019-12-23)
+    - .NET Core 3.0 (Microsoft support ended 2020-03-03)
+    - .NET Core 3.1
```

Comment:
```suggestion
    - .NET Core 3.1
    - .NET 5
```

We're recently added support in the .NET Tracer for .NET 5, the newest version of the ".NET Core" product